DepthChart/src/DepthChart.cpp

```diff
@@ -104,11 +104,15 @@ bool DepthChart::storeColor( Pixel pixel )
 		if( curr_accumulator.g > 1.0 ) curr_accumulator.g = 1.0;
 		if( curr_accumulator.b > 1.0 ) curr_accumulator.b = 1.0;
 
-		accumulator[key] += curr_accumulator;
+		if( curr_accumulator.r < 0.1 ) curr_accumulator.r = 0.1;
+		if( curr_accumulator.g < 0.1 ) curr_accumulator.g = 0.1;
+		if( curr_accumulator.b < 0.1 ) curr_accumulator.b = 0.1;
+
+		accumulator[key] = curr_accumulator;
```

Comment:
This was a critical bug, sent astronomical color numbers to Writer. += twice 
🤦‍♂